internal-employee-directory/README.md

```diff
@@ -3,9 +3,17 @@
 ## Overview
 This Internal Employee Directory is a modern, full-stack web application designed to manage and display employee information within an organization. It leverages a MongoDB database for efficient data storage, Node.js and Express.js for a robust server-side environment, and React.js with Tailwind CSS for an interactive and responsive client-side experience.
 
+## Table of contents
+- [TODO](#todo)
+- [Features](#features)
+- [Technologies Used](#technologies-used)
+- [High Level Design](#high-level-design)
+- [Tech Stack](#tech-stack)
+- [Installation](#installation)
+
+
 ## TODO:
 1. Writing unit test
-2. dockerizing front end
 3. CI CD pipeline
```

Comment:
nit, but fixup the numbering a cool thing you can do with markdown is to leave all the numbers as 1. then the engine will put in order, e.g. 
```
1. this
1. that
1. the other thing...
```
turns into
1. this
1. that
1. the other thing

Reply:
Ok, will do that